stackslib/src/net/stream.rs

```diff
@@ -0,0 +1,444 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
+// Copyright (C) 2020-2023 Stacks Open Internet Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::io;
+use std::io::{Read, Write};
+
+use stacks_common::types::chainstate::BlockHeaderHash;
+use stacks_common::types::chainstate::StacksBlockId;
+
+use crate::burnchains::Txid;
+use crate::chainstate::stacks::{StacksBlock, StacksBlockHeader, StacksMicroblock};
+
+use crate::chainstate::stacks::db::StacksChainState;
+use crate::chainstate::stacks::Error as chainstate_error;
```

Comment:
Same comment as elsewhere -- I know that this happens in other places, but it'd be good to avoid adding new instances of structs with camel_case.

```suggestion
use crate::chainstate::stacks::Error as ChainstateError;
```

Reply:
Yup